rules/windows/defense_evasion_scheduledjobs_at_protocol_enabled.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to enable the Windows Scheduled Jobs AT protocol. Attackers may use this method to move laterally or
+persist locally via the AT Jobs Protocol, which has been deprecated since Windows 8 and Windows Server 2012.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Scheduled Jobs AT Protocol Enabled"
```

Comment:
```suggestion
name = "Scheduled Tasks AT Command Enabled"
```

Reply:
not really command that get enabled, but easier to understand for users.